tests/sqllogictests/suites/query/02_function/02_0000_function_aggregate_state.test

```diff
@@ -1,9 +1,9 @@
 query T
 select length(max_state(number)), typeof(max_state(number)) from numbers(100);
 ----
-3 VARCHAR
+10 VARCHAR
```

Comment:
Does this means the state is larger than before?

Reply:
Yes, because we are using fixed-size encoding rather than variant encoding.

But this may be trivial, the protocol compression could be effective in the cluster.

Reply:
yes, that's the string length.

Reply:
> Yes, because we are using fixed-size encoding rather than variant encoding.
> 
> But this may be trivial, the protocol compression could be effective in the cluster.

More concerned about the new state memory usage, the OOM issue.

Reply:
serialization is only used in cluster queries.